src/objective_coefficients.jl

```diff
@@ -26,9 +26,9 @@ struct PrimalObjective{T}
 
     function PrimalObjective{T}(obj::SAF{T}) where T
         canonical_obj = MOIU.canonical(obj)
-        if isempty(canonical_obj.terms)
-            error("Dualization does not support models with no variables in the objective function.")
-        end
+        # if isempty(canonical_obj.terms)
```

Comment:
Add a comment explaining why this is commented, you can maybe just reference the Github discussion